velox/exec/RowContainer.h

```diff
@@ -437,6 +462,37 @@ class RowContainer {
     }
   }
 
+  template <TypeKind Kind>
+  static void extractColumnOffsetsTyped(
```

Comment:
nit: perhaps, replace Offsets with Rows for clarity

Reply:
Done.